### PR TITLE
website favicon icon changes

### PR DESCRIPTION
**Pull request by Ayushpanditmoto**
## 🛠️ Fixes Issue : ""
1. I have Added the Favicon Icon

Screenshot_3

### CODE REVIEW

**Comment by rezwancs** [comment]
@Ayushpanditmoto thanks for your effort🤩
Could you please do the below things ?
1. Please remove the 'package-lock.json' file from your PR.
2. Please include the issue number in your PR comment

Please don't forget to star this repository.

**Comment by Ayushpanditmoto** [comment]
okay I have remove the 'package-lock.json' file from my PR.
 Issues #1 @rezwan2525

File: src/components/Card.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
-import React from 'react';
+import React from 'react'
 
-export default function Card({id, front, back}) {
-
```
Comment (rezwancs) [inline src/components/Card.js:4]: You didn't work on this component. So, you should not push code changes for this inside this PR.

Comment (rezwancs) [reply src/components/Card.js:4]: @Ayushpanditmoto Please have a look here.

**Review by rezwancs — CHANGES_REQUESTED** [review]
@Ayushpanditmoto I meant to remove code changes of "package-lock.json" file from your PR (not completely delete the file)  by "Remove from PR"

File: src/components/Card.js (hunk 1 above)
Comment (Ayushpanditmoto) [reply src/components/Card.js:4]: fixed the issues now  no problem i hope so

**Pull request closed without merging**